build.gradle

```diff
@@ -245,7 +245,7 @@ dependencies {
     }
     compile 'org.jooq:jool-java-8:0.9.14'
 
-    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '6.14.2'
+    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.0.0.3'
```

Comment:
Feels a bit odd that first release of this new repository is at major version 7 ?